network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/DanglingLineImpl.java

```diff
@@ -437,4 +442,47 @@ public Optional<TieLine> getTieLine() {
         return Optional.ofNullable(resource.getAttributes().getTieLineId())
                 .flatMap(index::getTieLine);
     }
+
+    private <E extends Extension<DanglingLine>> E createOperatingStatusExtension() {
```

Comment:
As this is an Identifiable extension, it would be simpler (and less duplicated) to move it to AbstractIdentifiableImpl ?

Reply:
I did it at first to avoid duplication but reading at the code I realized that in other places, extensions are implemented at the equipment level:
* ActivePowerControl extends Injection and is implemented only in BatteryImpl and GeneratorImpl,
* IdentifiableShortCircuit extends Identifiable and is implemented only in VoltageLevelImpl. The code in IdentifiableShortCircuitImpl seems to be expecting VoltageLevel though.

As per our discussion, operatingStatus does not have much sense at the moment for the equipment types different than the one I implemented. On the other side, the equipment needs to also implement OperatingStatusHolder anyway to really support the extension, so it can be a safeguard. 
The only thing I see it's that we will have identifiable.getExtensions() returning operatingStatus in equipments that do not really support it while for now we did not introduce this case. I'm not sure it makes much sense?